### ISSUE TITLE
Upon Tardis reaching zero health, there's no interior effects

### ISSUE THREAD

**Issue by Chronosmage** [MattJeanes/TARDIS#493]
Hey 

Just thought upon Tardis destruction there's some exterior smoke effects but the interior just powers down, 

I was thinking if its possible the interior (Atm this is the default interior I've tested this with) should be shaking and on fire? As if the internal dimensions are breaking down with the loss of hull integrity.  (Atm this is the default interior I've tested this with) 

Also i recall the legacy version (which I know is vastly different to this version) Upon Tardis reaching zero health it could still Fly although it would be erratic with smoke everywhere

**Comment by MattJeanes** [comment]
Agreed, @parar020100 what kind of effects do you think we should have?

I don't believe this is a regression from the public workshop release

**Comment by parar020100** [comment]
I think there could be something like fire or smoke, but for a small amount of time
This actually correlates with #371, with "dead" and "severely broken" states, and if that's added, there could be a bit of fire and/or smoke in "severely broken"

**Comment by MattJeanes** [comment]
Yeah sounds good to me
